homeassistant/components/zha/light.py

```diff
@@ -114,6 +116,8 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
 class BaseLight(LogMixin, light.LightEntity):
     """Operations common to all light entities."""
 
+    _force_on = False
```

Comment:
This probably should be in `__init__()` since there's a setter for it

Reply:
I’m going to undo all of the last commit since it doesn’t work